### PR TITLE
[FIX] Placeholder labels appearing when changing servers

### LINKED ISSUE #1239: [BUG] Sometimes we're seeing placeholder labels user starts the app on left-sidebar menu

### PR DESCRIPTION
**Pull request by rafaelks**
@RocketChat/ios

Closes #1239

### CODE REVIEW

File: Rocket.Chat.xcodeproj/xcshareddata/xcschemes/Rocket.Chat.xcscheme [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0900"
+   LastUpgradeVersion = "0920"
```
Comment (rafaelks) [inline Rocket.Chat.xcodeproj/xcshareddata/xcschemes/Rocket.Chat.xcscheme]: This is just an Xcode suggestion.

File: Rocket.Chat/Models/Auth.swift [hunk 2]
```diff
@@ -40,7 +40,7 @@ final class Auth: Object {
 
     var user: User? {
         guard let userId = userId else { return nil }
-        return self.realm?.object(ofType: User.self, forPrimaryKey: userId)
+        return Realm.shared?.object(ofType: User.self, forPrimaryKey: userId)
```
Comment (rafaelks) [inline Rocket.Chat/Models/Auth.swift]: @cardoso Is there any reason to use `self.realm?` here?

Comment (cardoso) [reply Rocket.Chat/Models/Auth.swift]: It was probably to avoid the Singleton pattern.
Was it causing the problem?

**Comment by rafaelks** [comment]
Merging it before CI (is taking too long now, did run tests locally).

**Pull request merged**